modules/microshift-storage-about-vol-snapshots.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// microshift/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="about-volume-snapshots_{context}"]
+= About volume snapshots
+
+Volume snapshots can be used with LVM thin volumes to help protect against data loss in {product-title}. Only the LVMS CSI, `microshift csi`, is supported.
+
+[NOTE]
+====
+LVMS only supports the `WaitForFirstConsumer` `volumeBindingMode`, which means the storage volume is not provisioned until a pod is ready to mount it.
+====
+
+.Example workload that deploys a single pod and PVC
+
+[source, terminal]
+----
+oc apply -f - <<EOF
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-claim-thin
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: topolvm-provisioner-thin
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: base
+spec:
+  containers:
+  - command:
```

Comment:
This command does not look right. Surely there's an entrypoint in the container image to start the web server? @copejon can you help with that?